service/src/main/java/bio/terra/externalcreds/models/DistributedLock.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.externalcreds.models;
+
+import java.sql.Timestamp;
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface DistributedLock {
```

Comment:
```suggestion
public interface DistributedLock extends WithDistributedLock {
```

This, along with `./gradlew assemble`, will let you uncomment the builder.

Reply:
Oh! I had to add `extends WithDistributedLock` first and then run `./gradlew assemble` in order to generate that interface

Reply:
Hmm I'm still getting an error about the builder being final 🤔

Reply:
Oh `./gradlew assemble` fixed the error!